### ISSUE TITLE
Sequence train validation set loss extremely large

### ISSUE THREAD

**Issue by bwee** [jbohnslav/deepethogram#39]
Hi,  

Do you have any suggestions on how I can improve the sequence trainer?  The feature extractor is doing great, but the sequence trainer validation set loss keeps increasing instead of decreasing. 

Thanks a ton!

Feature extractor loss
![image](https://user-images.githubusercontent.com/12634702/107287570-bce00980-6a27-11eb-8b96-fe64bfea5900.png)

Sequence train loss
![image](https://user-images.githubusercontent.com/12634702/107287799-121c1b00-6a28-11eb-9db9-29c384d6ad48.png)

I used these configuration parameters in project_config.yaml:

`sequence:
  arch: tgmj
  bidirectional: false
  c_in: 1
  c_out: 8
  dropout_p: 0.5
  filter_length: 15
  hidden_dropout: 0.0
  hidden_size: 64
  input_dropout: 0.4
  latent_name: resnet18
  n_filters: 8
  nonlinear_classification: false
  nonoverlapping: false
  num_features: 64
  num_layers: 1
  output_name: null
  rnn_style: lstm
  sequence_length: 180
  soft_attn: false
  tgm_reduction: max
  weights: null`

Sequence train loss after inferring features a second time
![image](https://user-images.githubusercontent.com/12634702/107287634-d719e780-6a27-11eb-9a8d-39b405077dd3.png)

**Comment by jbohnslav** [comment]
Hi! Thanks for the question. This is actually kind of expected-- the sequence model has very few parameters, so it trains super fast--within the first epoch, which is why the learning curve looks so weird. Everything after that is just "overfitting." Note that it should definitely kill the job sooner than 60 epochs. Did you set the patience very high yourself?

The accuracy for the sequence model in this plot is just a bit better than the feature extractor. Hopefully other metrics (e.g. F1) are also better for the sequence model. In general, the sequence outputs are also smoother, which make them more similar to humans on things like bout statistics (figure 4 of the paper) than the feature extractor model.
